webapp/tests/Unit/Controller/API/JudgehostControllerTest.php

```diff
@@ -44,14 +44,14 @@ public function provideSingle(): Generator
     }
 
     /**
-     * Test that the endpoint returns an empty list for objects that don't exist
+     * Test that the endpoint returns an empty list for objects that don't exist.
      *
      * @dataProvider provideSingleNotFound
      */
     public function testSingleNotFound(string $id): void {
         $id = $this->resolveReference($id);
         $url = $this->helperGetEndpointURL($this->apiEndpoint, $id);
         $object = $this->verifyApiJsonResponse('GET', $url, 200, $this->apiUser);
-        static::assertEquals($object, []);
+        static::assertEquals([], $object);
```

Comment:
To me it reads nicer that object (the thing you worked on) is equal to something (the empty list), but this is probably convention?

Reply:
indeed, it's convention: https://phpunit.de/manual/6.5/en/appendixes.assertions.html#appendixes.assertions.assertEquals